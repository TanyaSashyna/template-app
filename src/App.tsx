import React, { FC } from "react";

const App: FC = () => (
  <div>App</div>
);

App.displayName = "app";

export default App;
