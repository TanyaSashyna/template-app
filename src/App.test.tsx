import React from "react";
import { cleanup, render } from "@testing-library/react";
import App from "./App";

afterEach(cleanup);

describe("App", () => {
  it("renders and matches snapshot", () => {
    const { container, asFragment } = render(
      <App/>
    );
      expect(container).toBeDefined();
    expect(asFragment()).toMatchSnapshot();
  });
})
