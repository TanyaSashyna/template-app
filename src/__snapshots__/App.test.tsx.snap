// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders and matches snapshot 1`] = `
<DocumentFragment>
  <div>
    App
  </div>
</DocumentFragment>
`;
